sktime/transformations/panel/catch22.py

```diff
@@ -45,27 +63,28 @@ class Catch22(BaseTransformer):
     """
 
     _tags = {
-        "scitype:transform-input": "Series",
-        # what is the scitype of X: Series, or Panel
+        "scitype:transform-input": "Panel",
```

Comment:
I don't think that's right - one feature, one series

Reply:
Does that include multivariate series? i.e. dimension lengths (10, 4, 20) would output (10, 88) by default. Can change it back.